model/project.rb

```diff
@@ -38,4 +38,19 @@ def has_valid_payment_method?
   def path
     "/project/#{ubid}"
   end
+
+  def current_invoice
+    begin_time = invoices.first&.end_time || Time.new(Time.now.year, Time.now.month, 1)
+    end_time = Time.now
+    content = {
```

Comment:
It would be better to populate this only if invoice is nil

Reply:
I moved it